frontend/index.html

```diff
@@ -3,79 +3,131 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/layout.css">
```

Comment:
1. rather than referencing all individual css files, a nicer approach is: import only one css file, e.g. `orders101-ui.css` and in that css file, import all the files you have created.  If there is only 1 css file to include in the markup you are less likely to forget one, it will be easier to manage.
1. look into and apply the "cache-breaker": use of a `#` in the css <link> so that the browser doesn't cache an old version of the css.

Reply:
Captured in [separate issue](https://github.com/MinhazH99/orders101/issues/46).